models/glif_psc.cpp

```diff
@@ -0,0 +1,578 @@
+/*
+ *  glif_psc.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "glif_psc.h"
+
+// C++ includes:
+#include <limits>
+#include <iostream>
+
+// Includes from libnestutil:
+#include "numerics.h"
+#include "propagator_stability.h"
+
+// Includes from nestkernel:
+#include "exceptions.h"
+#include "kernel_manager.h"
+#include "universal_data_logger_impl.h"
+#include "name.h"
+
+// Includes from sli:
+#include "dict.h"
+#include "dictutils.h"
+#include "doubledatum.h"
+#include "integerdatum.h"
+#include "lockptrdatum.h"
+
+using namespace nest;
+
+nest::RecordablesMap< nest::glif_psc > nest::glif_psc::recordablesMap_;
+
+namespace nest
+{
+// Override the create() method with one call to RecordablesMap::insert_()
+// for each quantity to be recorded.
+template <>
+void
+RecordablesMap< nest::glif_psc >::create()
+{
+  insert_( names::V_m, &nest::glif_psc::get_V_m_ );
+  insert_( names::ASCurrents_sum, &nest::glif_psc::get_ASCurrents_sum_ );
+  insert_( names::I_syn, &nest::glif_psc::get_I_syn_ );
+}
+}
+
+/* ----------------------------------------------------------------
+ * Default constructors defining default parameters and state
+ * ---------------------------------------------------------------- */
+
+nest::glif_psc::Parameters_::Parameters_()
+  : G_( 9.43 )                                           // in nS
+  , E_L_( -78.85 )                                       // in mV
+  , th_inf_( 27.17 )                                     // in mv, rel to E_L_, 51.68 - E_L_
```

Comment:
You can make this explicit as
```C++
, th_inf_( 51.68 - E_L_ )
```
since `E_L_` comes before `th_inf_` in the initializer list.